massmailer/models.py

```diff
@@ -370,6 +391,12 @@ class BatchEmail(models.Model):
         settings.AUTH_USER_MODEL, null=True, on_delete=models.SET_NULL
     )
 
+    content_type = models.ForeignKey(
+        ContentType, on_delete=models.CASCADE, null=True, blank=True
+    )
+    object_id = models.PositiveIntegerField(null=True, blank=True)
+    mailed_object = GenericForeignKey('content_type', 'object_id')
```

Comment:
what's the drawback of adding a field for the unsubscribe url instead of the model ? I don't like the GenericForeignKey API

Reply:
Please see IRC discussion, this will most likely be deleted anyway.